### PR TITLE
Translation support

### PR DESCRIPTION
**Pull request by stijnhau**
Translation support for this module.

### CODE REVIEW

File: language/de_DE.po [hunk 1]
```diff
@@ -0,0 +1,56 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: HtProfileImage\n"
+"Report-Msgid-Bugs-To: zf-devteam@zend.com\n"
```
Comment (ojhaujjwal) [inline language/de_DE.po]: @stijnhau  You missed it here!

Comment (stijnhau) [inline language/de_DE.po]: Fixed in new commit.
Strange that that field isn't show in poedit.

**Comment by ojhaujjwal** [comment]
Okay, I will test it and if it works, I will merge it!

**Comment by stijnhau** [comment]
Oke.
Any idea when?(just to have some idea)

**Comment by ojhaujjwal** [comment]
@stijnhau Is it working? I tried it!, But, all the lines are not being translated!

**Comment by stijnhau** [comment]
witch language?
Normaly it shoudl wortk for the dutch language (because i speak that one).
The other languages only have the strings translated that also are in ZfcUser because i cant translate those.
The mo and po filles are added and can be translated later on.

**Comment by ojhaujjwal** [comment]
Only the dutch language is working!
In that case, please remove other language which are not working!
If a guy wants another language, and he can send PR, then I will be happy to accept the PR!

**Comment by stijnhau** [comment]
The work but only the files that where in ZfcUser also.
I've think it is best if i let it so because in that case the string thatb where used and transledt by zfcUser also gets used in the view.
Else if we anable the module there are suddnely less strings translated.

**Pull request merged**